src/cdk/overlay/overlay-ref.ts

```diff
@@ -127,7 +131,8 @@ export class OverlayRef implements PortalHost {
    * Returns an observable that emits when the backdrop has been clicked.
    */
   backdropClick(): Observable<void> {
-    return this._backdropClick.asObservable();
+    return this._backdropInstance ? this._backdropInstance._clickStream.asObservable() :
+        empty<void>();
```

Comment:
Since the function return type is `Observable<void>`, you don't really need the `asObservable` call